Content.Shared/Chemistry/Reaction/ChemicalReactionSystem.cs

```diff
@@ -307,15 +312,10 @@ public void FullyReactSolution(Solution solution, EntityUid owner, FixedPoint2 m
     /// <reamrks>
     ///     Some solution containers (e.g., bloodstream, smoke, foam) use this to block certain reactions from occurring.
     /// </reamrks>
-    public sealed class ReactionAttemptEvent : CancellableEntityEventArgs
+    public partial record struct ReactionAttemptEvent(ReactionPrototype Reaction, Entity<SolutionComponent> Solution)
```

Comment:
Since DataDefinitions must be partial and most of the things I interact with are either DataDefinition or EntitySystem I have started to make everything I implement partial on reflex.